sys/arm64/arm64/copyinout.S

```diff
@@ -224,3 +224,128 @@ ending:
 	mov	x0, xzr		/* return 0 */
 	ret
 	.size	copycommon, . - copycommon
+
+#if __has_feature(capabilities)
+/*
+ * Copy specified amount of data from the user to kernel space, preserving
+ * capability tags
+ *
+ * int copyincap(const void * __capabilityuaddr, void *kdaddr, size_t len)
```

Comment:
```suggestion
 * int copyincap(const void * __capability uaddr, void *kdaddr, size_t len)
```

But just changing the comment is the easy part; the code actually needs to use the capabilities everywhere here.